.ci/Jenkinsfile

```diff
@@ -88,6 +88,10 @@ pipeline {
         }
         stage('Release') {
           options { skipDefaultCheckout() }
+          environment {
+            GOROOT = "${env.WORKSPACE}/.gimme/versions/go${env.GO_VERSION}.linux.amd64"
```

Comment:
I'm a bit skeptical of declaring those env variables explicitly, could we use a docker container instead?